scalableminds/webknossos
# Create org from register form and register from onboarding

Show a link to the onboarding flow when registering and `allowOrganizationCreation` is set.
Also allow to join an existing organization in the first onboarding step which will redirect the user to the register page with the organization pre-selected. This makes sense, because the `Get Started` button on instances with `allowOrganizationCreation` set is very eye-catching, so users may click it, although they actually want to simply register.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Set `allowOrganizationCreation` to true in the `application.conf`.
  - Visit the registration page, a link to the onboarding flow should be displayed.
  - In the first step of the onboarding flow:
    - Select an existing organization and click "Join" - You should be redirected to the registration page with the org prefilled.
    - Insert a new organization and click "Create" - The onboarding flow should continue as usual.

### Issues:
- fixes #3742 

------
